index.html

```diff
@@ -24542,13 +24542,39 @@ <h2 id="acknowledgments">
 
 
 <section class="appendix" id="revision_log">
-  <h2>Revision Log</h2>
+  <h2>
+  <span its-locale-filter-list="en" lang="en">Revision Log</span>
+  <span its-locale-filter-list="ja" lang="ja">変更記録</span>
+  </h2>
   <p its-locale-filter-list="en" lang="en">The following changes were made since the previous publication.</p>
+  <p its-locale-filter-list="ja" lang="ja">前回の公開時から以下の変更が行われています．</p>
   <ul>
-    <li id="id692">TBD</li>
+    <li>
+      <p its-locale-filter-list="en" lang="en">Merged and simplified <a href="#id237">handling of overhanging ruby text on punctuation marks</a> (former 3.3.8 f,g,h)</p>
+      <p its-locale-filter-list="ja" lang="ja"><a href="#id237">約物にかかるルビ文字の定義</a>を集約し簡単化しました (以前の 3.3.8 f,g,h)</p>
+      [<a href="https://github.com/w3c/jlreq/issues/15">Issue</a>, 
+      <a href="https://github.com/w3c/jlreq/pull/44">PR</a>]</li>
+    <li>
+      <p its-locale-filter-list="en" lang="en">English wording fixes</p>
+      <p its-locale-filter-list="ja" lang="ja">英語表現の修正</p>
+      [<a href="https://github.com/w3c/jlreq/issues/10">Issue</a>, 
+      <a href="https://github.com/w3c/jlreq/pull/66">PR</a>;
+      <a href="https://github.com/w3c/jlreq/issues/67">Issue</a>, 
+      <a href="https://github.com/w3c/jlreq/pull/82">PR</a>;
+      <a href="https://github.com/w3c/jlreq/issues/85">Issue</a>, 
+      <a href="https://github.com/w3c/jlreq/pull/91">PR</a>]</li>
+    <li>
+      <p its-locale-filter-list="en" lang="en">Merging English and Japanese version</p>
+      <p its-locale-filter-list="ja" lang="ja">英語版と日本語版の統合</p>
+      [<a href="https://github.com/w3c/jlreq/commit/7dcc9274b108ee14cafdc2b5041975cc277725ad">7dcc927</a>;
+      <a href="https://github.com/w3c/jlreq/issues/60">Issue</a>, 
+      <a href="https://github.com/w3c/jlreq/pull/88">PR</a>]</li>
+    <li>
+      <p its-locale-filter-list="en" lang="en">Link targets are assigned to each list item and note, it is possible to point into the document in a more fine-grained way.</p>
```

Comment:
were assigned, and there is some text missing from the sentence. I suggest:

> Link targets were assigned to each list item and note, making it possible to point into the document in a more fine-grained way.

Reply:
modified.